snap/snapcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: kubelogin # you probably want to 'snapcraft register <name>'
```

Comment:
should this be azure-kubelogin to avoid name collision?

Reply:
No, just kubelogin, as the idea is this is the official package, thus named 'kubelogin'. There is currently no other package named kubelogin, this one would be it.